src/main.c

```diff
@@ -212,18 +294,39 @@ void WriteInterfaces()
         IfNameLength = 0;
 
         switch (Interface->Type) {
-        case PCAPNG_LINKTYPE_ETHERNET:
-            printf("IF: medium=eth  ID=%u\tIfIndex=%u", Interface->PcapNgIfIndex, Interface->LowerIfIndex);
-            StringCchPrintfA(IfName, IF_STRING_MAX_SIZE, "eth:%lu", Interface->LowerIfIndex);
-            break;
-        case PCAPNG_LINKTYPE_IEEE802_11:
-            printf("IF: medium=wifi ID=%u\tIfIndex=%u", Interface->PcapNgIfIndex, Interface->LowerIfIndex);
-            StringCchPrintfA(IfName, IF_STRING_MAX_SIZE, "wifi:%lu", Interface->LowerIfIndex);
-            break;
-        case PCAPNG_LINKTYPE_RAW:
-            printf("IF: medium=mbb  ID=%u\tIfIndex=%u", Interface->PcapNgIfIndex, Interface->LowerIfIndex);
-            StringCchPrintfA(IfName, IF_STRING_MAX_SIZE, "mbb:%lu", Interface->LowerIfIndex);
-            break;
+            case PCAPNG_LINKTYPE_ETHERNET:
```

Comment:
You've added stray indentation here and elsewhere- please fix.

Reply:
fixed